go-apps/meep-metrics-engine/api/swagger-v2.yaml

```diff
@@ -0,0 +1,790 @@
+        }"
+      clientCorrelator: "83"
+      subscriptionId: "1.2556487E7"
+  EventSubscriptionResponseList:
+    properties:
+      eventSubscriptionResponse:
+        type: "array"
+        items:
+          $ref: "#/definitions/EventSubscriptionResponse"
+      resourceURL:
+        type: "string"
+        format: "uri"
+        example: "http://localhost:8291/v2/subscriptions/event/subscription123"
+        description: "Self referring URL."
+    example:
+      resourceURL: "http://localhost:8291/v2/subscriptions/event/subscription123"
+      eventSubscriptionResponse:
+      - resourceURL: "http://localhost:8291/v2/subscriptions/events/subscription123"
+        eventQueryParams:
+          scope:
+            duration: "10s"
+            limit: 60
+          fields:
+          - "event"
+          tags:
+          - name: "type"
+            value: "MOBILITY"
+        callbackReference: "{\"notifyURL\":\"http://clientApp.example.com/metric_event_notifications/123456\"\
+          }"
+        clientCorrelator: "83"
+        subscriptionId: "1.2556487E7"
+      - resourceURL: "http://localhost:8291/v2/subscriptions/events/subscription123"
+        eventQueryParams:
+          scope:
+            duration: "10s"
+            limit: 60
+          fields:
+          - "event"
+          tags:
+          - name: "type"
+            value: "MOBILITY"
+        callbackReference: "{\"notifyURL\":\"http://clientApp.example.com/metric_event_notifications/123456\"\
+          }"
+        clientCorrelator: "83"
+        subscriptionId: "1.2556487E7"
+  Events_callbackReference:
+    type: "object"
+    required:
+    - "notifyURL"
+    properties:
+      notifyURL:
+        type: "string"
+        format: "url"
+        example: "http://clientApp.example.com/metric_event_notifications/123456"
+        description: "The URL of your own listener application."
+    description: "Notification callback definition."
+    example: "{\"notifyURL\":\"http://clientApp.example.com/metric_event_notifications/123456\"\
+      }"
+  NetworkSubscriptionParams:
+    type: "object"
+    properties:
+      clientCorrelator:
+        type: "string"
+        example: "83"
+        description: "Uniquely identifies this create subscription request. If there\
+          \ is a communication failure during the request, using the same clientCorrelator\
+          \ when retrying the request allows the operator to avoid creating a duplicate\
+          \ subscription."
+      callbackReference:
+        $ref: "#/definitions/Network_callbackReference"
+      networkQueryParams:
+        $ref: "#/definitions/NetworkQueryParams"
+      recurrence:
+        type: "integer"
+        example: 1
+        description: "Recurrence of recurring-time-based notifications"
+      subscriptionType:
+        type: "string"
+        example: "Recurring-time-based"
+        description: "Type of subscription triggering notifications"
+        enum:
+        - "Recurring-time-based"
```

Comment:
Rename property: updateInterval

Description: "Notification interval in seconds, disabled if set to 0"

Reply:
fixed